src/test/java/FavoritesTests.java

```diff
@@ -0,0 +1,81 @@
+import POM.pages.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class FavoritesTests extends BaseTest{
+
+    @Test
+    public void emptyFavorites(){
+        HomePage homePage = new LoginPage(driver).signIn("nozishka86@gmail.com","te$t$tudent");
```

Comment:
**Please, remove code redundancies!**

**Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )**

![image](https://user-images.githubusercontent.com/104761482/205431045-0b36a6db-5158-4d51-b0cf-ae4e846fc95a.png)

_For example_

```
    @BeforeMethod
    public void sigIn(){
        HomePage homePage = new LoginPage(driver).signIn("nozishka86@gmail.com","te$t$tudent");
   }
```